proxy/src/main/java/org/openjdk/skara/proxy/HttpProxy.java

```diff
@@ -49,49 +55,50 @@ public static void setup() {
 
             var output = new String(p.getInputStream().readAllBytes(), StandardCharsets.UTF_8).trim();
             var res = p.waitFor();
-            if (res == 0 && output != null && !output.isEmpty()) {
-                if (output.startsWith("https://") || output.startsWith("http://")) {
-                    var uri = new URI(output);
-                    setProxyHostAndPortBasedOn(uri);
-                } else {
-                    for (var scheme : List.of("http", "https")) {
-                        var uri = new URI(scheme + "://" + output);
-                        setProxyHostAndPortBasedOn(uri);
-                    }
+            if (res == 0 && !output.isEmpty()) {
+                if (!output.startsWith("http://") && !output.startsWith("https://")) {
+                    output = "http://" + output;
```

Comment:
Add a comment that the protocol being prepended doesn't matter?

Reply:
Sure, sounds reasonable.